src/Host/Client/Impl/BrokerServices/RemoteUriWebService.cs

```diff
@@ -55,7 +55,7 @@ public class RemoteUriWebService : IRemoteUriWebService {
             var remoteUri = GetRemoteUri(context.Request.Url, remoteBaseUrl);
             request.Headers.Add(CustomHttpHeaders.RTVSRequestedURL, remoteUri.ToString());
 
-            if (context.Request.InputStream.CanSeek && context.Request.InputStream.Length > 0) {
+            if (context.Request.ContentLength64 > 0) {
```

Comment:
Opening the stream when there is not supposed to be content will lead to error. So, the request stream has to be accessed only if there is content or in websocket context.